v1/models/base.py

```diff
@@ -165,6 +178,20 @@ class Date(BaseModel):
     )
 
 
+class PostcodeLocation(BaseModel):
+    type: str
+    properties: Optional[
+        Dict[str, Any]
+    ]  # Change to Optional[Dict[str, Any]] if properties are dynamic
```

Comment:
This comment is confusing because it says "change to `xyz`" but the line it is commenting is already `xyz`.

Reply:
Copy and paste speed run busted! Fixed up :)